tests/test_templates_layers.py

```diff
@@ -19,9 +19,7 @@
 import pytest
 import pennylane as qml
 from pennylane import numpy as np
-from pennylane.templates.layers import (CVNeuralNetLayers,
-                                        StronglyEntanglingLayers,
-                                        RandomLayers)
+from pennylane.templates.layers import CVNeuralNetLayers, StronglyEntanglingLayers, RandomLayers
```

Comment:
Most of the changes are due to a run of `black`. I've highlighted the actual addition below.

Reply:
Ah, this is because the test suite is currently unblacked 🙂